Sources/Public/Animation/AnimationView.swift

```diff
@@ -784,14 +784,14 @@ final public class AnimationView: LottieView {
       let positionAnimation = animation.copy() as? CABasicAnimation ?? CABasicAnimation(keyPath: "position")
       positionAnimation.keyPath = "position"
       positionAnimation.isAdditive = false
-      positionAnimation.fromValue = animationLayer.position
+      positionAnimation.fromValue = animationLayer.presentation()?.position
```

Comment:
It may make sense to handle the case where `presentation()` is `nil`, thoughts?

```suggestion
      positionAnimation.fromValue = (animationLayer.presentation() ?? animationLayer).position
```

Reply:
I went ahead and made this change in https://github.com/airbnb/lottie-ios/pull/1395/commits/3b57a052226915417ca27862a7758a363a40a74f